sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/MetricDataMapper.java

```diff
@@ -273,10 +273,10 @@ private static boolean isSuccess(String metricName, Long statusCode, boolean cap
     }
 
     private static boolean isClient(String metricName) {
-       return metricName.contains(".client.");
+        return metricName.contains(".client.");
```

Comment:
looks like we missed one space. would be nice to have `spotlessApply` to handle this automatically. I know this is in the roadmap. do you know when that will happen for `sdk/monitor/*`?

Reply:
That can be added any time when we feel comfortable with Spotless formatting the code in Monitor OpenTelemetry Exporter. I can file another PR after this for that if you want.

Reply:
sooner the better. we use spotless everywhere from java agent to otel upstream. it's a handy tool for us.